fern/api/generators.yml

```diff
@@ -1,4 +1,14 @@
 groups:
+  local:
+    generators:
+      - name: fernapi/fern-go-sdk
+        version: 0.3.0
+        config:
+          module:
+            path: github.com/hookdeck/hookdeck-go-sdk
+        output:
+          location: local-file-system
+          path: ../../local/generated/hookdeck-go-sdk
```

Comment:
i assume we can add this `local` group and it won't affect the Fern release CI, just confirming that this PR is safe to merge